src/main/resources/messages_en.properties

```diff
@@ -23,6 +23,7 @@ buq.error.invalid.format.uuid=UUID {0} parameter has wrong format for field {1}.
 buq.error.invalid.format.boolean=Boolean {0} parameter has wrong format for field {1}.
 buq.error.invalid.searchParams=To search bottom-up quantification you can provide only program parameters.
 buq.error.invalid.parameter.bottomUpQuantificationStatus=Parameter "status" is not a valid Bottom-Up Quantification Status: {0}.
+buq.error.periodAndFacilityUnique=Selected period is already prepared
```

Comment:
I would suggest changing the message for the user to be more accurate, e.g. "The bottom-up quantification for the selected processing period has already been created."